src/LiveComponent/assets/src/Component/ValueStore.ts

```diff
@@ -41,19 +41,10 @@ export default class {
     get(name: string): any {
         const normalizedName = normalizeModelName(name);
 
-        if (this.dirtyProps[normalizedName] !== undefined) {
-            return this.dirtyProps[normalizedName];
-        }
-
-        if (this.pendingProps[normalizedName] !== undefined) {
-            return this.pendingProps[normalizedName];
-        }
-
-        if (this.props[normalizedName] !== undefined) {
-            return this.props[normalizedName];
-        }
-
-        return getDeepData(this.props, normalizedName);
+       return this.dirtyProps[normalizedName] ||
+               this.pendingProps[normalizedName] ||
+               this.props[normalizedName] ||
+               getDeepData(this.props, normalizedName);
```

Comment:
Is this form of if statement precisely the same as before? For example, if a value is defined, but is null, previously, it would return null. Will it now?